aspnetcore/toc.md

```diff
@@ -6,7 +6,6 @@
 ## [ASP.NET Core tutorials](xref:tutorials/index)
```

Comment:
Whenever you see this Tutorials node because you have Getting Started expanded, you also see the main Tutorials node immediately following.  For that reason I would say just delete this one.